workspace/apps/pidp/src/app/features/access/pages/bc-provider-application/bc-provider-application.component.ts

```diff
@@ -28,14 +33,21 @@ export class BcProviderApplicationComponent
   implements OnInit
 {
   public faUser = faUser;
+  public faCircleCheck = faCircleCheck;
+  public faXmark = faXmark;
   public formState: BcProviderApplicationFormState;
   public showErrorCard = false;
   public errorCardText = '';
   public showMessageCard = false;
   public messageCardText = '';
   public completed: boolean | null;
+  public username = '';
   public password = '';
-  public showOverlayOnSubmit = true;
+  public showOverlayOnSubmit = false;
+
+  @ViewChild('successDialog')
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  public successDialogTemplate!: TemplateRef<any>;
```

Comment:
```suggestion
  public successDialogTemplate!: TemplateRef<Element>;
```
You can use the generic interface `Element` for an HTML element and remove the comment above